### PR TITLE
Stringsupdate

### PR DESCRIPTION
**Pull request by lachlan-00**
Scrutinizer changes as well as the massive stringsupdate from #1957 

Please merge when you can also update Transifex with a squash and merge.

### CODE REVIEW

File: lib/class/api.class.php [hunk 1]
```diff
@@ -1966,7 +1966,7 @@ public static function user_update($input)
         $password   = $input['password'];
         $state      = $input['state'];
         $city       = $input['city'];
-        $disable    = ($input['disable'] == 'true' ? true : false;
+        $disable    = $input['disable'] == 'true' ? true : false;
```
Comment (Psy-Virus) [inline lib/class/api.class.php]: It's always best practice not to mix coding styles. You always used braces `( ... )` around inline comparisons. Why not here?
In my opinion it also makes it easier to read.

**Comment by Psy-Virus** [comment]
Hey @lachlan-00, what exactly do you mean by

> Please merge when you can also update Transifex with a squash and merge.

?

Transifex updates the source strings by itself once a day. Then, if changes happen, the translators can start working on them and I pull the changes from Transifex and merge them here. 

I could ofc cancel Transifex from updating the source, upload the new source file manually and we can wait for the Translators to finish. But I think the way we do it usually is better, as people can see the strings in context of the UI and stuff. Even if that means that we have an not completely translated release.

:)

File: lib/class/api.class.php (hunk 1 above)
Comment (lachlan-00) [reply lib/class/api.class.php]: They do. and that's changed. added a few missing ones as well.

**Comment by lachlan-00** [comment]
> Hey @lachlan-00, what exactly do you mean by
> 
> > Please merge when you can also update Transifex with a squash and merge.
> 
> ?
> 
> Transifex updates the source strings by itself once a day. Then, if changes happen, the translators can start working on them and I pull the changes from Transifex and merge them here.
> 
> I could ofc cancel Transifex from updating the source, upload the new source file manually and we can wait for the Translators to finish. But I think the way we do it usually is better, as people can see the strings in context of the UI and stuff. Even if that means that we have an not completely translated release.
> 
> :)

Good to know! I always assumed that you had to manually sync with transifex which is why i assigned this to you

**Comment by kuzi-moto** [comment]
Can't believe it's finally merged!

**Comment by lachlan-00** [comment]
It's time!

Now it's time to bug fix and document!

**Comment by stebe** [comment]
Great work on this update. I'm impressed on how bug free after playing around different menus and functions for the last 45 minutes. Now doing a catalog update on my largest catalog.

**Comment by lachlan-00** [comment]
Really Appreciated! There's some bug fixes in there for you @stebe.

The branch was my daily driver for about a month so it's had the time to settle. :)

**Comment by Psy-Virus** [comment]
> 
> 
> > Hey @lachlan-00, what exactly do you mean by
> > > Please merge when you can also update Transifex with a squash and merge.
> > 
> > 
> > ?
> > Transifex updates the source strings by itself once a day. Then, if changes happen, the translators can start working on them and I pull the changes from Transifex and merge them here.
> > I could ofc cancel Transifex from updating the source, upload the new source file manually and we can wait for the Translators to finish. But I think the way we do it usually is better, as people can see the strings in context of the UI and stuff. Even if that means that we have an not completely translated release.
> > :)
> 
> Good to know! I always assumed that you had to manually sync with transifex which is why i assigned this to you

Well I guess I had a brain lag yesterday... 
So how it's working is:
I update the messages.pot -> PR it -> after merge Transifex is pulling it and updating the sources in Transifex itself. And about once a month I pull the stuff from Transifex and PR it here.

So ehm.... GTO is coming in a few minutes. Then the translators can hack the new translations in... 😁

**Comment by lachlan-00** [comment]
Okay! I think i'll write this down in the wiki so we remember

**Pull request merged**